pkgs/top-level/all-packages.nix

```diff
@@ -11065,6 +11065,8 @@ in
 
   uhd = callPackage ../applications/radio/uhd { };
 
+  uhhyou.lv2 = callPackage ../applications/audio/uhhyou.lv2 { };
```

Comment:
That's a terrible name for an attribute at `top-level/all-packages.nix`. How about `LV2Plugins`? I noticed there are other lv2 plugins derivations (I don't know what that is) with similar names. Perhaps you should build yourselves a mini "ecosystem" of `lv2Plugins` or `lv2.pkgs` attribute set, where they can be composed together like Python... However, that's out of scope for this PR so perhaps this name is not too bad.

Reply:
I agree a category for lv2 plugins might be useful.
To name this package LV2Plugins, even though tha is the name of the upstream repo, would be verry odd. It would be a good name for the category, not for one small set of plugins.

Reply:
Have you seen https://github.com/NixOS/nixpkgs/pull/89547?

Reply:
> Have you seen #89547?

No! (:clap: to @FRidh for taking over this!) @magnetophon it'll be helpful if you'd wait for upstream to respond to the issues we presented to them regarding this plugin and help @FRidh test their changes at #89547 . Afterwards, I'd suggest posting a diff adding this plugin to to #89547 and ask @FRidh to apply it as well.